repair_shards.h

```diff
@@ -20,11 +20,11 @@
 /* templates for SQL commands used during shard placement repair */
 #define DROP_REGULAR_TABLE_COMMAND "DROP TABLE IF EXISTS %s"
 #define DROP_FOREIGN_TABLE_COMMAND "DROP FOREIGN TABLE IF EXISTS %s"
+#define COPY_RELATION_QUERY "SELECT worker_copy_shard_placement(%s, %s, %d)"
```

Comment:
I guess I was distinguishing between commands (that can modify) and queries (that can't). But that's a false distinction… this `SELECT` (a "query") is making modifications by calling a funtion (so it's a "command"). I'll go with `COPY_SHARD_PLACEMENT_COMMAND`